client/src/pages/mainlayout/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { MuiThemeProvider, CssBaseline } from "@material-ui/core";
+import { BrowserRouter } from "react-router-dom";
+import { theme } from "../../themes/theme";
```

Comment:
I don't like using relative paths that need to go up directories (like this, using ..)
Take a look at https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d

Reply:
Updated to not use relative paths.